swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+        A vendor at a swap meet. 
+
+        Attributes:
+            inventory (list): a list of Item objects (default: [])
+    '''
+    def __init__(self, inventory=None):
+        self.inventory = inventory
+        if self.inventory is None:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+        except ValueError:
+            return False
+        return item
+    
+    def get_by_category(self, category):
+        items_in_category = [item for item in self.inventory 
+                            if item.category == category]
+        return items_in_category
+
+    def swap_items(self, vendor, my_item, their_item):
+        #this if statement calls .remove regardless of outcome;
+        #will cause item removal if present in inventory
+        if not (vendor.remove(their_item)):
+            return False
+        if not (self.remove(my_item)):
+            vendor.add(their_item)
+            return False
+
+        vendor.add(my_item)
+        self.add(their_item)
+        return True
```

Comment:
Good callout in the comments! One tiny style note: the parentheses on lines 32 and 34 are not needed.